node/src/chain_spec/oak.rs

```diff
@@ -196,6 +197,115 @@ pub fn oak_staging() -> ChainSpec {
 	)
 }
 
+pub fn oak_live() -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), TOKEN_SYMBOL.into());
+	properties.insert("tokenDecimals".into(), TOKEN_DECIMALS.into());
+	properties.insert("ss58Format".into(), SS_58_FORMAT.into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"OAK Network",
+		// ID
+		"oak",
+		ChainType::Live,
+		move || {
+			let allocation_json = &include_bytes!("../../../distribution/oak_alloc.json")[..];
+			let initial_allocation: Vec<(AccountId, Balance)> =
+				serde_json::from_slice(allocation_json).unwrap();
+			const ALLOC_TOKENS_TOTAL: u128 = DOLLAR * 1_000_000_000;
+			validate_allocation(
+				initial_allocation.clone(),
+				ALLOC_TOKENS_TOTAL,
+				EXISTENTIAL_DEPOSIT,
+			);
+
+			let vesting_json = &include_bytes!("../../../distribution/oak_vesting.json")[..];
+			let initial_vesting: Vec<(u64, Vec<(AccountId, Balance)>)> =
+				serde_json::from_slice(vesting_json).unwrap();
+
+			let vested_tokens = 9_419_999_999_999_999_919;
+			let vest_starting_time: u64 = 1669874400;
+			let vest_ending_time: u64 = 1764568800;
+			validate_vesting(
+				initial_vesting.clone(),
+				vested_tokens,
+				EXISTENTIAL_DEPOSIT,
+				vest_starting_time,
+				vest_ending_time,
+			);
+
+			testnet_genesis(
```

Comment:
make sure to update this wihtt he correct collators

Reply:
where do i find the right collator addresses?